zoo-css-challenge/style.css

```
@@ -160,19 +199,34 @@ footer {
   background-size: cover;
   background-position: center;
   padding: 100px 5%;
+  
 }
 
 .education-container {
   background-color: rgba(255, 255, 255, 0.75);
   padding: 50px;
   max-width: 400px;
+  border-radius: 25px;
 }
 
 .education-container h3,
 .education-container p {
   margin-bottom: 1em;
 }
 
+.education-container button{
+  font-size: larger;
+
+}
+
+.education-container button:hover{
+  background-color: blueviolet;
+} 
+
+.education-container button:focus{
+  background-color: blueviolet;
+} 
```

```suggestion
.education-container button:hover,
.education-container button:focus{
  background-color: blueviolet;
} 
```
We can combine the two selectors as they both have the same CSS declaration

---

Ok, we can by comma, Thank you very much